src/distributing-policies/secure-supply-chain.md

```diff
@@ -0,0 +1,387 @@
+# Secure Supply Chain
+
+Users can configure Kubewarden to run policies signed by only entities that they
+trust by using the [sigstore](https://www.sigstore.dev) project. This means that
+policy developers can sign their policies and publish them in a registry.
+The cluster operators can run these signed policies and ensure the validity of the policies chain of custody
+
+
+## Signing the policies
+
```

Comment:
Maybe it would be a good idea to explain what signing entails: that it will add a layer into the container image, and get it pushed into the repo. Then, when people sign things later, they know what to expect.

Reply:
we could also explain how to show the signatures with `kwctl inspect`